test/test_data.py

```diff
@@ -163,6 +163,7 @@ def testall(self):
             parser = self.parsers[td['parser']]
             test = getattr(module, td['class'])
 
+            description = ''
```

Comment:
If self.silent is True, then lines 196 and 197 fail as description is undefined.